### DISCUSSION TITLE
How to configure node labels/roles/tains?

### DISCUSSION THREAD

**Discussion by CRASH-Tech** [siderolabs/sidero#1053] [Q&A] [answered]
How to configure it without kubectl after cluster creation?

**Comment by smira** [comment, marked as answer]
https://www.talos.dev/v1.3/reference/configuration/#machineconfig see `nodeLabels`

**Reply by CRASH-Tech** [reply]
> https://www.talos.dev/v1.3/reference/configuration/#machineconfig see `nodeLabels`

Thanks, but where it in the cluster-api config? (also in the previous question)

**Reply by smira** [reply]
https://www.sidero.dev/v0.5/guides/patching/
